bin/smack.py

```diff
@@ -140,6 +198,14 @@ def arguments():
   verifier_group.add_argument('--smackd', action="store_true", default=False,
     help='generate JSON-format output for SMACKd')
 
+  svcomp_group = parser.add_argument_group('svcomp options')
+
+  svcomp_group.add_argument('--svcomp', action="store_true", default=False,
```

Comment:
I suppose we could switch to a custom front end front end for `.i` files, however this wouldn't eliminate the need for `--svcomp`, as indicated in the above comment.

Also, I feel this wouldn't be the cleanest solution, anyway.  A `.i` extension indicates that the file has already been preprocessed, not that it is an svcomp benchmark.  It is conceivable that a user may want to verify a preprocessed file outside of the context of svcomp.

With that said, it *would* be nice to have some sort of detection feature to switch on svcomp mode...